cmake doesnt check for `protoc`

A recent support request on the discord revealed the fact that cmake doesn't currently check for the existence of a `protoc` executable. cmake succeeds without error, including finding libprotobuf, but a subsequent `make` results in an error:
```
make[2]: *** No rule to make target '../common/pb/Protobuf_PROTOC_EXECUTABLE-NOTFOUND', needed by 'common/pb/session_event.pb.cc'.  Stop.
CMakeFiles/Makefile2:177: recipe for target 'common/pb/CMakeFiles/cockatrice_protocol.dir/all' failed
make[1]: *** [common/pb/CMakeFiles/cockatrice_protocol.dir/all] Error 2
Makefile:149: recipe for target 'all' failed
make: *** [all] Error 2
```

This can occur when protobuf is split into multiple packages, including a separate one for protoc, like `protobuf-compiler` on debian.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -163,6 +163,9 @@ set(CMAKE_AUTOMOC TRUE)
 
 # Find other needed libraries
 FIND_PACKAGE(Protobuf REQUIRED)
+IF(NOT EXISTS "${Protobuf_PROTOC_EXECUTABLE}")
+  MESSAGE(FATAL_ERROR "No protoc command found!")
+ENDIF()
 
 #Find OpenSSL
 IF(WIN32)

